### DISCUSSION TITLE
Meo IPTV

### DISCUSSION THREAD

**Discussion by ramoluis** [fabianishere/udm-iptv#119] [Q&A]
HI.  Has anyone successfully tested IPTV MEO with UDM-PRO?

**Comment by ghost** [comment]
@ramoluis did you found a solution ?

**Reply by ramoluis** [reply]
yes, I have a UDM-PRO working with a MEO box.

**Reply by ghost** [reply]
Can you please help me ? I've got an UDM PRO for several months but now I want to use the meo box. 

Do you created the IpTV vlan or are you using the untaged lan instead ? 

Can you please tell me how your infrastructure are ?

Thank you in advance

**Reply by ramoluis** [reply]
I created vlan for tv meo.  When I get back home I can provide a little tutorial I made of my setup.

**Reply by ramoluis** [reply]
[MEO_IPTV_v2.pdf](https://github.com/fabianishere/udm-iptv/files/10369178/MEO_IPTV_v2.pdf)
This was the configuration I made. It has been working for over 6 months.
Good luck

**Reply by ramoluis** [reply]
@jresende7, you have already successfully tested

**Reply by ghost** [reply]
> @jresende7, you have already successfully tested

Thank you very much for your help and the tutorial you sent 

But unfortunately I couldn't do the way you said in the tutorial, and the image still stops after 10s. 

Maybe the igmp proxy is not working properly.

**Reply by pedromrg** [reply]
@ramoluis great tutorial, I have followed it and it's alsmot working 100%, still I got an issue: #233 I can watch for 5-15min and I have a freeze... not sure what could be happening
